README.md

```diff
@@ -7,38 +7,62 @@
 </a>
 
 **rust-code-analysis** is a Rust library to analyze and extract information
-from source codes written in many different programming languages.
+from source code written in many different programming languages.
 It is based on a parser generator tool and an incremental parsing library
 called
 <a href="https://tree-sitter.github.io/tree-sitter/" target="_blank">Tree Sitter</a>.
 
 
-In addition, we provide a command line tool called **rust-code-analysis-cli**
-to interact with the API of the library in an easy way.
+A command line tool called **rust-code-analysis-cli** is provided to interact with the API of the library in an easy way.
 
 This tool can be used to:
 
 - Call **rust-code-analysis** API
 - Print nodes and metrics information
 - Export metrics in different formats
 
-# Software Usability
+In addition, we provide a **rust-code-analysis-web** tool to use the library through a REST API.
+
+
+# Usage
 
 **rust-code-analysis** supports many types of programming languages and
-computes a great variety of metrics. If you want to discover more on this
-software, read our
+computes a great variety of metrics. You can find up to date documentation at
 <a href="https://mozilla.github.io/rust-code-analysis/index.html" target="_blank">Documentation</a>.
 
 On the
 <a href="https://mozilla.github.io/rust-code-analysis/commands/index.html" target="_blank">
     Commands
-</a> page, you will learn which commands need to be run to get information
+</a> page, there is a list of commands that can be run to get information
 about metrics, nodes, and other general data provided by this software.
 
+
+# Contributing
+
 If you want to contribute to the development of this software
 or you are just interested in building **rust-code-analysis**, have a look at the
 <a href="https://mozilla.github.io/rust-code-analysis/developers/index.html" target="_blank">Developers Guide</a>.
 
+
+# How to cite rust-code-analysis
+
+```
+@article{ARDITO2020100635,
+    title = {rust-code-analysis: A Rust library to analyze and extract maintainability information from source codes},
+    journal = {SoftwareX},
+    volume = {12},
+    pages = {100635},
+    year = {2020},
+    issn = {2352-7110},
+    doi = {https://doi.org/10.1016/j.softx.2020.100635},
+    url = {https://www.sciencedirect.com/science/article/pii/S2352711020303484},
+    author = {Luca Ardito and Luca Barbato and Marco Castelluccio and Riccardo Coppola and Calixte Denizet and Sylvestre Ledru and Michele Valsesia},
```

Comment:
Is it usual to use `and`, `and`, ... ?

Reply:
Yeah. I copied the BibTex from https://www.sciencedirect.com/science/article/pii/S2352711020303484.